utils/networks.ts

```diff
@@ -99,6 +99,21 @@ export const networks = {
       providers: [{ provider: getRpcUrl(polygon.id), priority: 1, weight: 1 }],
     },
   },
+  [arbitrum.id]: {
+    blockExplorerUrl: 'https://arbiscan.io/',
+    railgunNetworkName: NetworkName.Arbitrum,
+    chainId: arbitrum.id,
+    wethAddress: '0x82aF49447D8a07e3bd95BD0d56f35241523fBab1',
+    baseToken: {
+      symbol: 'AETH',
```

Comment:
Arbitrum docs + popular apps all use `ETH` now (used to be AETH a while ago)
```suggestion
      symbol: 'ETH',
```